Hi @quark2 there are Clan warnings to fix:
DQM/GEM/plugins/GEMDQMHarvester.cc
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_0_X_2020-02-17-1100/src/DQM/GEM/plugins/GEMDQMHarvester.cc:31:8: warning: 'dqmEndLuminosityBlock' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]

--- a/DQM/GEM/plugins/GEMDQMHarvester.cc
+++ b/DQM/GEM/plugins/GEMDQMHarvester.cc
@@ -31,7 +31,7 @@ protected:
   void dqmEndLuminosityBlock(DQMStore::IBooker &,
                              DQMStore::IGetter &,
                              edm::LuminosityBlock const &,
-                             edm::EventSetup const &);
+                             edm::EventSetup const &) override;
   void dqmEndJob(DQMStore::IBooker &, DQMStore::IGetter &) override{};  // Cannot use; it is called after dqmSaver
 
   void refineSummaryHistogram(edm::Service<DQMStore> &);

